### PR TITLE
feat: add feature to enlarge image on click

### PR DESCRIPTION
**Pull request by anudeep-bpgc**
Signed-off-by: Anudeep Ghosh <91572535+anudeep-bpgc@users.noreply.github.com>

# Description

This PR adds the feature requested under the issue id #3089 for allowing users to enlarge images by clicking on them
in the documentation.
Dependencies include usage of JavaScript library called ['medium-zoom'](https://www.npmjs.com/package/medium-zoom)

Fixes # (issue)
[/layouts/partials/scripts.html](https://github.com/jenkins-x/jx-docs/blob/main/layouts/partials/scripts.html)
added medium-zoom JavaScript library source from CDN
added JavaScript code using HTML Element selector to select images from documentation and add the zoom-in functionality

[/assets/scss/_code.scss](https://github.com/jenkins-x/jx-docs/blob/main/assets/scss/_code.scss)
added code to keep the zoomed-in image above nav-bar

# Checklist:

- [x] I have mentioned the appropriate type(scope), as referenced [here](https://jenkins-x.io/community/code/#the-commit-message) in the commit message and PR title for the semantic checks to pass.
- [x] I have signed off the commit, as per instructions mentioned [here](https://jenkins-x.io/community/code/#how-to-sign-your-commits).
- [x] Any dependent changes have already been merged.

### CODE REVIEW

File: assets/scss/_code.scss [hunk 1]
```diff
@@ -61,3 +61,7 @@
         padding: $pre-padding;
     }
 }
+
+.medium-zoom-overlay, .medium-zoom-image--opened {
+    z-index: 2000 !important;
+}
```
Comment (ankitm123) [inline assets/scss/_code.scss:67]: NIT: remove new line.

**Review by ankitm123 — COMMENTED** [review]
minor comment

**Comment by ankitm123** [comment]
Very nice, verified that it works on the images. Just one minor comment, and then it's good to merge.

**Comment by anudeep-bpgc** [comment]
Hi @ankitm123 , 
Thanks for the code review.
I just have a small doubt, your minor comment asks for removal of the new line, whereas, already no newline is present at the end of the file. What do I need to do?

File: assets/scss/_code.scss (hunk 1 above)
Comment (ankitm123) [reply assets/scss/_code.scss:67]: There seems to be an extra line after 67, but it's non blocking. I will approve. Tthis is good stuff ...

**Review by ankitm123 — APPROVED** [review]

File: assets/scss/_code.scss (hunk 1 above)
Comment (anudeep-bpgc) [reply assets/scss/_code.scss:67]: Thank you so much for the PR merge

**Pull request merged**